packages/KMenu/KMenuItem.vue

```diff
@@ -0,0 +1,156 @@
+<template>
+  <div
+    :data-testid="item ? `${item.title.replace(' ', '-')}-menu-item` : 'menu-item'"
+    :test-mode="testMode"
+    :class="[isOpen && expandable ? 'title-dark' : '', {'expand-item' : expandable}]"
+    class="k-menu-item">
+    <div
+      class="menu-title"
+      @click="isOpen = !isOpen">
```

Comment:
<!-- Issues represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**issue:** ‏we should not have click events on non clickable elements. This should be an <a> or <button>

Reply:
<!-- Issues represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**issue:** ‏any click event should also have a keyboard equivalent

Reply:
can we add aria attributes for this

aria-controls
aria-expanded

Reply:
So this click event needs to be removed ?

Reply:
we can have the click event, but the element should be changed to a button

Reply:
for the keyboard event see https://github.com/maranran/eslint-plugin-vue-a11y/blob/master/docs/rules/click-events-have-key-events.md

Reply:
Okay! I see. I will add button and keyboard equivalent and aria attributes.